index.js

```diff
@@ -1,45 +1,162 @@
 const binding = require('./binding')
 const { isNode } = require('which-runtime')
 
-module.exports = binding
+const api = { ...binding }
+module.exports = api
```

Comment:
```suggestion
module.exports = exports = { ...binding }
```

Then we can remove `api` as well and just use `exports`.

Reply:
I did that intentionally to reduce verbosity on js side and for clarity when a wrapper forwards a call to another wrapper compared to the native binding.  (ref: crypto_generichash_batch)